css/global.css

```diff
@@ -1,7 +1,8 @@
 li.jobs-unified-top-card__job-insight--highlight,
 div.jobs-unified-top-card__primary-description > div > span:last-child,
 li.job-card-container__footer-item > strong,
-li.job-card-container__applicant-count
+li.job-card-container__applicant-count,
+.jobs-unified-top-card__applicant-count
```

Comment:
This is the class being used for the applicant count at the top right of a job posting.